examples/README

```diff
@@ -1,33 +1,31 @@
+Examples
+--------
```

Comment:
Suggestion, use `# Examples` for consistency.

Reply:
This is on purpose and not the same as #.